site/content/integrations/grafana.md

```diff
@@ -3,19 +3,60 @@ title: "Grafana"
 weight: 3
 ---
 
+## Overview 
 
-M3 supports a variety of Grafana integrations.
+Grafana is an open-source visualization tool used to dashboarding, querying, and monitoring databases, especially used for time series databases. See more [here](https://grafana.com/). 
 
-## Prometheus / Graphite Sources
+M3 supports Grafana integrations for both Prometheus and Graphite/StatsD metrics. It is recommeneded to use Prometheus metrics when integrating with M3DB. 
 
-M3Coordinator can function as a datasource for Prometheus as well as Graphite. See the [Prometheus integration](/docs/integrations/prometheus) and [Graphite integration](/docs/integrations/graphite) documents respectively for more information.
+## Getting started with Grafana 
 
-## Pre-configured Prometheus Dashboards
+Refer to Grafana's [Getting Started documention](https://grafana.com/docs/grafana/latest/getting-started/). 
+
+### Provisioning Grafana
+
+Learn how to provision your Grafana instances [here](https://grafana.com/docs/grafana/latest/administration/provisioning/#datasources). 
+
+### Datasources
+
+Grafana suports many different datasources. When using Grafana, you will be able to switch views across your datasources. Grafana includes built-in support for [Prometheus](https://grafana.com/docs/grafana/latest/datasources/prometheus/) and [Graphite](https://grafana.com/docs/grafana/latest/datasources/graphite/). 
+
+See a list of all supported datasources [here](https://grafana.com/docs/grafana/latest/datasources/).
+
+### Dashboarding 
```

Comment:
I feel like this whole section should be a bit more specific to M3, i.e., what are best practices for dashboards when using M3.